app/controllers/application_controller.rb

```diff
@@ -107,7 +107,7 @@ def api_deprecation_error(exception = nil)
   def smart_proxy_exception(exception = nil)
     process_error(:redirect => :back, :error_msg => exception.message)
     logger.warn "ProxyAPI operation FAILED: #{exception}"
-    logger.debug exception.backtrace.join("\n")
+    logger.warn exception.backtrace.join("\n")
```

Comment:
Could we make it a bit shorter? `Foreman::Log.exception` maybe? And one small comment, I'd like to have `e` as the first argument, context might be long message and the exception could be hard to find. Otherwise looking forward to have this available in Foreman.